apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -44,6 +66,63 @@ export const askAITutor = createAsyncThunk(
   }
 );
 
+// General Chat
+// This thunk's callback function submits a user chat message to the chat completion endpoint,
+// waits for a chat completion response, and updates the user message state.
+export const submitChatMessage = createAsyncThunk(
+  'aitutor/submitChatMessage',
+  async (message: string, thunkAPI) => {
+    const state = thunkAPI.getState() as {aiTutor: AITutorState};
+    const systemPrompt =
```

Comment:
nit: This is just a system prompt for general chat, right? Should we call it `generalChatSystemPrompt`? It might also be useful at some point to put all these hard-coded system prompts and initial messages into an aiTutor constants file so it's easy for us to go back and find the things we need to un-hardcode, but we don't need to do that in this PR.